src/tools/kvstore_tool.h

```diff
@@ -62,6 +62,9 @@ class StoreTool
 	   ceph::bufferlist& val);
   bool rm(const std::string& prefix, const std::string& key);
   bool rm_prefix(const std::string& prefix);
+  void print_summary(const uint64_t total_keys, const uint64_t total_size,
+                     const uint64_t total_txs, const string &store_path,
```

Comment:
ditto.

Reply:
Bah! Must be a clang-format thing... using LLVM style...